src/mame/igs/igs_fear.cpp

```diff
@@ -11,94 +11,139 @@
 #include "screen.h"
 #include "speaker.h"
 
-
 namespace {
 
 class igs_fear_state : public driver_device
 {
 public:
-	igs_fear_state(const machine_config &mconfig, device_type type, const char *tag)
-		: driver_device(mconfig, type, tag),
+	igs_fear_state(const machine_config &mconfig, device_type type, const char *tag) :
+		driver_device(mconfig, type, tag),
 		m_maincpu(*this, "maincpu"),
-		m_gfxdecode(*this, "gfxdecode"),
-		m_palette(*this, "palette") { }
+		m_videoram(*this, "videoram"),
+		m_palette(*this, "palette"),
+		m_gfxrom(*this, "gfx1")
+	{ }
 
 	void igs_fear(machine_config &config);
 
 	void init_igs_fear();
+	void init_igs_superkds();
 
 protected:
 	virtual void video_start() override;
 
 private:
-	void sound_irq(int state);
-	uint32_t screen_update_igs_fear(screen_device &screen, bitmap_ind16 &bitmap, const rectangle &cliprect);
-	void pgm_create_dummy_internal_arm_region();
 	required_device<cpu_device> m_maincpu;
-	optional_device<gfxdecode_device> m_gfxdecode;
+	required_shared_ptr<u32> m_videoram;
 	required_device<palette_device> m_palette;
+	required_region_ptr<uint8_t> m_gfxrom;
+
 	void igs_igs_fear_map(address_map &map);
```

Comment:
`igs_fear_state::igs_igs_igs_igs_igs_igs` ...

Reply:
you're moaning about code that hasn't even changed now?

is driving away new contributors not enough for you?

Reply:
I see your name on top of driver, I'm stating the fact that you are violating conventional naming here. Enjoy the collision.

Reply:
again, you're moaning about code that hasn't even changed here

the last time you moaned because somebody had done cleanups at the same time

it's starting to feel like you just get off on moaning

Reply:
git disagrees with you, check below instead of wasting energy in ranting here.

Reply:
Just fix it to "fear_map" and get rid of the unconditional logerror() in draw_sprite() and it's good to go.